src/components/App.tsx

```diff
@@ -2597,10 +2597,11 @@ class App extends React.Component<ExcalidrawProps, AppState> {
 
         const allHitElements = pointerDownState.hit.allHitElements;
         const hitElement = pointerDownState.hit.element;
-        const isHittingOnlyOneElementAndItIsNotSelected =
-          !this.isASelectedElement(hitElement) && allHitElements.length === 1;
+        const someHitElementIsSelected = allHitElements.some((element) =>
+          this.isASelectedElement(element),
+        );
         if (
-          (hitElement === null || isHittingOnlyOneElementAndItIsNotSelected) &&
```

Comment:
unclear why only one element needed to be hit

Reply:
I reviewed that bit of code to make sure. It doesn't. I wasn't thinking clearly at the time I wrote that. Thank you for pointing it out :)